lib/attrsets.nix

```diff
@@ -1798,6 +1798,43 @@ rec {
       then pkg.${output} or pkg.out or pkg
       else pkg;
 
+  /**
+    Like `getOutput` but with a list of fallback output names.
+    This function is alligned with `_overrideFirst()` from the `multiple-outputs.sh` setup hook.
+
+    # Inputs
+
+    `outputs`
+
+    : 1\. Function argument
+
+    `pkg`
+
+    : 2\. Function argument
+
+    # Type
+
+    ```
+    getOutput' :: [String] -> Derivation -> String
```

Comment:
Ah, also, this has long been a lie because of `... or pkg` which returns an attrset